.travis.yml

```diff
@@ -3,7 +3,7 @@ language: go
 branches:
   only:
   - gh-pages
-  - develop
+  - "/develop|release-*/"
```

Comment:
Also need to regex the tag branch name?

Reply:
You are right. Added support for release tag like `v0.1.0-rc.1`.